Kitodo/src/main/webapp/pages/searchResult.xhtml

```diff
@@ -17,7 +17,9 @@
         xmlns:h="http://xmlns.jcp.org/jsf/html"
         xmlns:f="http://xmlns.jcp.org/jsf/core"
         xmlns:p="http://primefaces.org/ui">
-
+    <f:metadata>
+        <f:viewParam name="id"/>
```

Comment:
I am a little confused here. How can the access rights to a process be determined on a per-process basis if you pass just one `id` parameter to this view that is than used in every row of the datatable? Shouldn't the ID instead be determined by each specific record of the datatables values, e.g. something like `process.id`?